www/source/docs/reference.html.md.erb

```diff
@@ -1239,6 +1239,188 @@ Packages are installed in the `/hab/pkgs/` directory, and then further organized
 **hooks directory**
 : If you have defined a `hooks` subdirectory with hook scripts in your plan, then they will be included in the same relative location within the installed package directory. Read more about [Application Lifecycle hooks](/docs/reference/#reference-hooks).
 
+---
+## <a name="supervisor-config" id="supervisor-config" data-magellan-target="supervisor-config"> Supervisor Configuration File</a>
+
+Generate a Supervisor configuration file use it to set any of the 40+ Supervisor configuration settings instead of configuring them on the command line.
+
+<div style="justify-content: center; display: flex; padding-top:2rem; padding-bottom:2rem">
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/B_ppJ1xaC5o" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen ></iframe>
+</div>
+
+Chef Habitat looks for Supervisor configuration settings in four locations and applies them in order, which means that settings defined in later locations overwrite earlier ones.
+
+```bash
+hab sup run --generate-config
+```
+
+```bash
+hab sup run --generate-config > config.toml
+```
+
+The Supervisor settings look-up order:
+
+1. The fixed config file location `/hab/sup/default/config/sup.toml` on the Supervisor
```

Comment:
"on the Supervisor" maybe "on the Supervisor machine" or some synonym (system, node)